cmd/atlas/internal/cmdapi/schema.go

```diff
@@ -108,6 +122,16 @@ func schemaApplyRun(cmd *cobra.Command, _ []string, flags schemaApplyFlags) erro
 		return err
 	}
 	defer client.Close()
+	var dev *sqlclient.Client
+	if flags.DevURL != "" {
+		dev, err := sqlclient.Open(cmd.Context(), flags.DevURL)
+		if err != nil {
+			return err
+		}
+		defer dev.Close()
+	} else {
+		dev = client
```

Comment:
I am not entirely sure I like this. The `Snapshoter` will guard us against accidently dropping any resource here, but I am not sure I like passing in the actual connection here. 

@a8m Thoughts on this?

Reply:
Ask for `--dev-url` in case it's needed. Unlike the `Normalizer` which operates on a temporary schema, we may run arbitrary commands on the connection (migration dir replay), and reverting changes is not enough, because we may delete data.